client-lite/src/download/download.cpp

```diff
@@ -311,9 +311,11 @@ void Download::_Abort() try
     _timer.Stop();
     _fileStream.Close();
     _CancelTasks();
-    if (!_destFilePath.empty())
+    // Delete file only if this download is the creator/owner. The abort could be from an
+    // error downloading to an already existing file.
+    if (_fDestFileCreated && !_destFilePath.empty())
     {
-        boost::filesystem::remove(_destFilePath);
```

Comment:
>        boost::filesystem::remove(_destFilePath) [](http://example.com/codeflow?start=0&length=48)

Unrelated to this change but part of cleaning up our dependencies, the change you introduced already helps us remove usage of boost::filesystem, so there's potential to see if we can just remove some of the boost libraries in enterity

Reply:
boost::filesystem is still used in other parts like the rest_port_advetiser and finder. The biggest boost library is boost.log so I'm thinking of looking into that after the cpprestsdk work item.